src/main/webapp/trickeditpage.html

```diff
@@ -10,9 +10,15 @@
       <div class="form-header">
         <h2>Add Trick:</h2>
       </div>
-      <form style="margin:10px;input" action="/timeline_data" method="POST">
+      <form style="margin:10px;input" action="/timeline_data" method="POST" onsubmit="return results();">
```

Comment:
Where is the results function?

Reply:
There is no js file linked to this html page, so how can it call a js function? I also don't see a function named results() anywhere

Reply:
Oh I forgot to update that. The functions is supposed to be called loadResults()